zmq/sugar/socket.py

```diff
@@ -677,7 +677,7 @@ def recv_json(self, flags=0, **kwargs):
     _poller_class = Poller
 
     def poll(self, timeout=None, flags=POLLIN):
-        """Poll the socket for events.
+        """Poll the socket for an event.
```

Comment:
Can we leave this plural? It is a mask reporting potentially multiple events, so it seems confusing to me to say "event" refers to one or more events that are ready.

Reply:
The [documentation](https://github.com/zeromq/pyzmq/blob/d28d7cae78a1292bf882924d757cdf884a61eef6/zmq/sugar/poll.py#L94) for Poller.poll() refers to this bitmask as `event`. It seems like these should be consistent, to reduce confusion. 

Would it be better to change the Poller.poll() to call this bitmask `events`?

Reply:
You're definitely right they should match! I do think plural makes more sense.

Reply:
I changed both to event_mask. Does that sound reasonable? When I first read the document, I thought events was a list of events. When I realized it was an int, I thought it referred to the number of events. 

I think we should make it really clear that this is actually the bitmap.

Reply:
Good idea, thanks!